### DISCUSSION TITLE
What is the minimum SEB version that supports bootstrap 5?

### DISCUSSION THREAD

**Discussion by kabirpathak** [SafeExamBrowser/seb-win-refactoring#605] [General]
- I've recently bumped the bootstrap version to v5 in my website.
  - While the changes are working on other browsers, they fail to work on SEB 2.4. 
  - The bootstrap specific attributes (data-bs-*) are not working. (modals, dropdowns, popovers, etc)
  - Which is the minimum SEB version that supports bootstrap 5?

- Also, rails-ujs specific code stopped working in SEB, after this upgrade in bootstrap version. However,
   - changing Rails.ajax to fetch api worked for me. Any insights on this?

Any infomation will be really helpful. Regards.

**Comment by dbuechel** [comment]
SEB 2.x isn't supported anymore, you should (for various important reasons) update to version 3.x as soon as possible.

**Reply by kabirpathak** [reply]
Does v3 support bootstrap5?

**Reply by dbuechel** [reply]
Yes, I don't see why SEB 3.x (which is using Chromium as browser engine) shouldn't be able to run a website built with Bootstrap 5.
